docs/quickstarts/openshift-cost-explorer/metadata.yml

```diff
@@ -0,0 +1,6 @@
+kind: QuickStarts # kind must always be "QuickStarts"
+name: openshift-cost-explorer
+tags: # If you want to use more granular filtering add tags to the quickstart
+  - kind: bundle
+    value: OpenShift
```

Comment:
this should be lowercase (unless the router does not care and it works anyways)

Reply:
@Hyperkid123 done. I was using the bundle that was already present in this repo for `openshift-related-documentation` (so I guess it would have worked?).

Reply:
I honestly don't know. The learning resources UI is configured to use the `openshift` value for the Opensihft page.